test_driver/isolates_workaround.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:async';
+
+import 'package:flutter_driver/flutter_driver.dart';
+
+/// Workaround for bug: https://github.com/flutter/flutter/issues/24703
+class IsolatesWorkaround {
+  IsolatesWorkaround(this._driver, {this.log = false});
+  final FlutterDriver _driver;
+  final bool log;
+  StreamSubscription _streamSubscription;
+
+  Future<void> resumeIsolates() async {
+    final vm = await _driver.serviceClient.getVM();
+    // Unpause any paused isolated
+    for (final isolateRef in vm.isolates) {
+      final isolate = await isolateRef.load();
+      if (isolate.isPaused) {
+        await isolate.resume();
+        if (log) {
+          print('Resuming isolate: ${isolate.numberAsString}:${isolate.name}');
+        }
+      }
+    }
+    if (_streamSubscription != null) {
+      return;
+    }
+    _streamSubscription = _driver.serviceClient.onIsolateRunnable
+        .asBroadcastStream()
+        .listen((isolateRef) async {
+      final isolate = await isolateRef.load();
```

Comment:
Extract these 7 lines into something like `_loadAndResumeIsolate()` since it's done here and above